viral/ensemble_reactivation.py

```
@@ -1257,20 +1554,22 @@ def all_mice_ensemble_results_plots(
         label="preactivation",
     )
     plt.xlabel("Time (s) from event onset")
+    1 / 0
```

Ruff (0.14.0)

Found useless expression. Either assign it to a variable or remove it.

(B018)